core/src/main/scala/cats/Applicative.scala

```diff
@@ -263,6 +263,9 @@ trait Applicative[F[_]] extends Apply[F] with InvariantMonoidal[F] { self =>
   def whenA[A](cond: Boolean)(f: => F[A]): F[Unit] =
     if (cond) void(f) else unit
 
+  def elseA[A](cond: Boolean)(f: => F[A])(other: => F[A]): F[A] =
+    if (cond) f else other
```

Comment:
This method does not seem doing anything with `F[_]` itself, so it looks like an implementation of a ternary operator, e.g.:
```
def x: F[A] = ???
def y: F[A] = ???

elseA(cond)(x)(y) <-> if (cond) x else y
```
In other words, this implementation does not seem related to `Applicative` (or any other typeclass in **cats**).

Besides, there's a convenience syntax in **mouse**:
```
import mouse.boolean.*

cond.fold(x, y)
```